src/app/dynamic/enterprise/quotas/quota-widget/template.html

```diff
@@ -0,0 +1,137 @@
+   CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+   TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+   SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+END OF TERMS AND CONDITIONS
+-->
+
+<div class="km-quota-widget"
+     *ngIf="isEnterpriseEdition && quotaDetails">
+  <ng-container *ngTemplateOutlet="showCompactView ? compactView : extendedView"></ng-container>
+</div>
+
+<ng-template #compactView>
+  <mat-card *ngIf="showInCard; else progressQuota">
+    <mat-card-content>
+      <ng-container *ngTemplateOutlet="progressQuota"></ng-container>
+    </mat-card-content>
+  </mat-card>
+
+  <ng-template #progressQuota>
+    <div fxLayout="row"
+         fxLayoutAlign="center center">
+      <i *ngIf="showIcon"
+         class="km-icon-quota i-32"></i>
+
+      <ng-container *ngIf="quotaDetails.quota.cpu"
+                    [ngTemplateOutlet]="progressBar"
+                    [ngTemplateOutletContext]="{label: 'CPU', percentage: quotaPercentage.cpu}">
+      </ng-container>
+
+      <ng-container *ngIf="quotaDetails.quota.memory"
+                    [ngTemplateOutlet]="progressBar"
+                    [ngTemplateOutletContext]="{label: 'Memory', percentage: quotaPercentage.memory}">
+      </ng-container>
+
+      <ng-container *ngIf="quotaDetails.quota.storage"
+                    [ngTemplateOutlet]="progressBar"
+                    [ngTemplateOutletContext]=" {label: 'Disk', percentage: quotaPercentage.storage}">
+      </ng-container>
+    </div>
+
+    <mat-card class="km-detail-view"
+              *ngIf="showDetails$ | async">
+      <mat-card-content fxLayout="column">
+        <div class="km-detail-row">Project Quota</div>
+
+        <div class="km-detail-row"
+             *ngIf="quotaDetails.quota.cpu">
+          <span>CPU</span>
+          <span>{{quotaDetails.status.globalUsage.cpu}}/{{quotaDetails.quota.cpu}}</span>
+        </div>
+
+        <div class="km-detail-row"
+             *ngIf="quotaDetails.quota.memory">
+          <span>Memory</span>
+          <span>{{quotaDetails.status.globalUsage.memory}}/{{quotaDetails.quota.memory}}</span>
+        </div>
+
+        <div class="km-detail-row"
+             *ngIf="quotaDetails.quota.storage">
+          <span>Disk</span>
+          <span>{{quotaDetails.status.globalUsage.storage}}/{{quotaDetails.quota.storage}}</span>
+        </div>
+      </mat-card-content>
+    </mat-card>
+
+    <ng-template #progressBar
+                 let-label="label"
+                 let-percentage="percentage">
+      <km-property class="km-quota-item"
+                   fxLayout="column"
+                   fxLayoutAlign="center center">
+        <div key>{{ label }}</div>
+        <mat-progress-bar value
+                          class="property-usage-bar"
+                          mode="determinate"
+                          [color]="getProgressBarAccent(percentage)"
+                          [value]="percentage ?? 1"
```

Comment:
why default value `1`? make it `0` i guess?

Reply:
default value is `1` because when it is `0` the progress bar doesn't show

Reply:
To add clarification further this is edge case when Quota is ser but `percentage` is `0` so then it make sense to have 1 to at-least show progress bar 👍🏽